bin/kafka-run-class.sh

```diff
@@ -140,6 +140,11 @@ do
 done
 shopt -u nullglob
 
+if [ -z "$CLASSPATH" ] ; then
+  echo 'Classpath is empty. Please build the project first e.g. by running ''./gradlew jar -Pscala_version=$SCALA_VERSION'''
```

Comment:
Yes, it doesn't print $SCALA_VERSION variable. I thought, that if this variable is unset, then message would be strange, with such command: `./gradlew jar -Pscala_version`. Do you suggest to set variable, if it's unset? I should use double quotes in `echo` then.